content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -19,8 +19,8 @@ We generate and process rich metadata that powers compiler-accurate code navigat
 
 ### Vision
 
-> In the near term we want Code intelligence to provide seamless, out-of-the-box, precise code navigation for languages that cover 90% of the market usage.
 > We see Code intelligence as the glue that sticks the product together, providing a platform for features from navigation to precise powered searches, compiler-accurate batch changes and insights. We aim to provide support for all widely used languages and for the ones we don’t, provide a platform for any developer to add and test their own indexers.
+> In the near term we want Code intelligence to provide seamless, out-of-the-box, precise code navigation for languages that cover 90% of the market usage.
 > We want our code navigation to reach IDE feature parity, while offering the option of plugging into developers’ favorite IDEs.
 > In the longer term, we envision building a global knowledge graph that accurately maps the entire code universe.
 
```

Comment:
In the guiding principles, the second one is great but I think the first one duplicates the Code Graph level principles so no need to repeat it here, unless you think it's specifically adding value.